src/database/sqlite/event/event_database.py

```diff
@@ -1784,6 +1782,7 @@ def _row_to_stored_pairing(cls, row: dict[str, Any]) -> StoredPairing:
             round_=row['round'],
             result=row['result'],
             board_id=row['board_id'],
+            illegal_moves=row.get('illegal_moves', 0),
```

Comment:
```suggestion
            illegal_moves=row['illegal_moves'],
```
We need thiss to fail if there's an issue with the migration, it will fail at another layer which will be harder to realize otherwise